src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1067,20 +1067,17 @@
      *
      * <p>For example:</p>
      * <pre>
-     * this(1);
+     * |--this -&gt; 1
```

Comment:
Please revert this change. `this(1);` is the expression parsed for this example.